ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -132,16 +144,21 @@ routes.register<typeof GetTransactionStreamRequestSchema, GetTransactionStreamRe
         const burns = new Array<RpcBurn>()
 
         for (const note of tx.notes) {
-          const decryptedNote = note.decryptNoteForOwner(request.data.incomingViewKey)
+          let decryptedNote = note.decryptNoteForOwner(request.data.incomingViewKey)
+
+          if (!decryptedNote && request.data.outgoingViewKey) {
+            decryptedNote = note.decryptNoteForSpender(request.data.outgoingViewKey)
+          }
 
           if (decryptedNote) {
             const assetValue = await node.chain.getAssetById(decryptedNote.assetId())
             notes.push({
               value: CurrencyUtils.encode(decryptedNote.value()),
-              memo: BufferUtils.toHuman(decryptedNote.memo()),
+              memo: decryptedNote.memo().toString('hex'),
```

Comment:
We could make this `utf8` to preserve some backwards compatibility and still avoid removing non-human-readable characters.

On the other hand, `hex` is consistent with how we treat all other buffers in the RPC